### ISSUE TITLE
kickoff

### ISSUE THREAD

**Issue by marcosci** [tsmcgrath/longdistancehikes#1]
Hi Tim! 

Nice to online-meet you 😄 

Should we start with the ones in the wikipedia link you shared and store them as geojson here?

Cheers
Marco

**Comment by tsmcgrath** [comment]
Yeah. Let's start with the 4 on your initial list and then move on from there. I think I have two and there are some answers from other repliers pointing to data.

**Comment by tsmcgrath** [comment]
Sorry, it's the program manager in me, I had to start a Kanban board to track to-do's. Mostly for my own tasks but feel free to use it.
https://github.com/tsmcgrath/longdistancehikes/projects/2

**Comment by marcosci** [comment]
Sure thing, keeping everything organised is probably a good idea!
I just uploaded the PCT - it s just the whole thing as a multilinestring. 

I think more metadata for the trails (shelter, elevation, whatnot) would just end up being messy for the beginning, or?
But - I haven't thought about it more than two minutes, so very happy for suggestions! 😄

**Comment by tsmcgrath** [comment]
We can have one info page per trail and if there is any related info we want to link to just put it in there for now. We can start with readme.md's and move to html if we want to expose it as a web site via Github pages.

**Comment by tsmcgrath** [comment]
I agree we should just focus on geometry for now.

**Comment by marcosci** [comment]
Could you write one for the PCT? Just that I know what you are thinking of :)

**Comment by tsmcgrath** [comment]
Sure. We should have source info for the geometry in there as well.

**Comment by tsmcgrath** [comment]
The PCT geometry I have is from 2015 from Halfmile (trailname) https://pctmap.net/ We'd probably have to get permission to expose. But, it's freely available on their site (as GPX) so it could be easy to get permission. What source is yours? Do you want the alternative segments as well?
